src/Core/Application/Service/ManifestValidator/ManifestValidator.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Service\ManifestValidator;
+
+use SprykerSdk\Sdk\Core\Application\Dependency\ManifestValidatorInterface;
+use Symfony\Component\Config\Definition\Processor;
+
+class ManifestValidator implements ManifestValidatorInterface
```

Comment:
`ManifestValidator` should be moved to the `infrastructure` layer. Also, we don't need an interface, because it always calls from the TaskYamlRepository.